### PR TITLE
always show the dismiss all button

### LINKED ISSUE #605: "dismiss all notifications" button should always be visible
### Steps to reproduce
1. have a lot of notifications
2. click on the bell icon

### Expected behaviour
when there are notifications, the "dismiss all notifications" button should always visible at the bottom of the notification window. When there are too many notifications, they should flow behind the "dismiss all notifications" button

### Actual behaviour
You see all the individual notifications but not a "dismiss all" button -  you think you have to "x" them all individually, or you scroll down by accident to discover the "dismiss all notifications" button. Bad UX.

### comments
related to https://github.com/nextcloud/notifications/issues/105

### Server configuration

**Nextcloud version:** 18.0.3

### PR DESCRIPTION
**Pull request by szaimen**
Close #605
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: css/styles.scss [hunk 1]
```diff
@@ -43,6 +43,8 @@
 	.notification-wrapper {
 		display: flex;
 		flex-direction: column;
+		margin-bottom: 44px;
+		max-height: calc(100vh - 50px * 4 - 44px) !important;
```
Comment (nickvergessen) [inline css/styles.scss:47]: lets hope no one uses a device that is smaller than 244 pixels + 50px of the header

**Review by nickvergessen — APPROVED** [review]

**Comment by szaimen** [comment]
Can we backport this at is a UX improvement?

**Comment by szaimen** [comment]
Thanks! :)

**Pull request merged**